grafana.yml

```diff
@@ -94,3 +116,34 @@
       with_items:
         - 80/tcp
         - 443/tcp
+    - name: Add on-call backend
+      when: grafana_on_call
+      block:
+        - name: Create dir
+          ansible.builtin.file:
+            path: "{{ grafana_on_call_path }}"
+            state: directory
+            owner: "{{ ansible_ssh_user }}"
+            group: "{{ ansible_ssh_user }}"
+          become: true
+        - name: Get docker compose
+          ansible.builtin.get_url:
+            url: https://raw.githubusercontent.com/grafana/oncall/49d20f1a7e40669e901db95443603138e1a1cde4/docker-compose.yml
```

Comment:
This compose file deploys Prometheus and a Grafana server, among other services. Do we want this? 

Maybe it would be ideal to have a trimmed version of this compose file?

Reply:
only if you set the according env variable:
~~~
echo "DOMAIN=http://localhost:8080
# Remove 'with_grafana' below if you want to use existing grafana
# Add 'with_prometheus' below to optionally enable a local prometheus for oncall metrics
# e.g. COMPOSE_PROFILES=with_grafana,with_prometheus
COMPOSE_PROFILES=with_grafana
# to setup an auth token for prometheus exporter metrics:
# PROMETHEUS_EXPORTER_SECRET=my_random_prometheus_secret
# also, make sure to enable the /metrics endpoint:
# FEATURE_PROMETHEUS_EXPORTER_ENABLED=True
SECRET_KEY=my_random_secret_must_be_more_than_32_characters_long" > .env
~~~
https://github.com/grafana/oncall/tree/dev?tab=readme-ov-file#getting-started

Reply:
Oh cool! Thanks for the clarification.